### ISSUE TITLE
`--platform` needs to be the first flag on the commandline, otherwise it is ignored

### ISSUE THREAD

**Issue by hash-d** [skupperproject/skupper#1488]
**Describe the bug**

If a different flag precedes `--platform` on the command line, the value given to it is ignored.

**How To Reproduce**

Try a `skupper init` with `--platform` not as the first flag:

```
$ skupper init --ingress-host localhost --platform podman --enable-console --enable-flow-collector --site-name skgw --console-auth=internal --console-user=admin --console-password=password
Secret already exists:  skupper-local-server
Secret already exists:  skupper-local-client
Secret already exists:  skupper-service-client
Secret already exists:  skupper-console-users
Secret already exists:  skupper-site-server
Secret already exists:  skupper-console-certs
Waiting for status...
Skupper is now installed in namespace 'dh-1248'.  Use 'skupper status' to get more information.
```

Notice that the command above installed skupper on the K8S namespace `dh-1248`, instead of the requested podman install.

See that changing the placement of the `--platform` flag causes the correct behavior to take place.

```
$ skupper init --platform podman --ingress-host localhost --enable-console --enable-flow-collector --site-name skgw --console-auth=internal --console-user=admin --console-password=password
Skupper is now installed for user 'dhashimo'.  Use 'skupper status' to get more information.
```

Also, note that the value of the `--platform` is completely ignored:

```
$ skupper init --ingress-host localhost --platform asdf --enable-console --enable-flow-collector --site-name skgw --console-auth=internal --console-user=admin --console-password=password
Secret already exists:  skupper-local-server
Secret already exists:  skupper-local-client
Secret already exists:  skupper-service-client
Secret already exists:  skupper-console-users
Secret already exists:  skupper-site-server
Secret already exists:  skupper-console-certs
Waiting for status...
Skupper is now installed in namespace 'dh-1248'.  Use 'skupper status' to get more information.
```

On the example above, the invalid value `asdf` was given to `--platform` and yet it was allowed to continue.

**Expected behavior**

Either: 

 - The value of `--platform` should be respected, regardless of its position on the command line, or
 - The need for `--platform` flag to be the first in the command line should be documented

I did not see anything on the documentation pointing to the necessity of `--platform` to be the first in the command line.  Even in that case, an error should be generated when that's not in its proper place, instead of continuing with a configuration different from what the user specified.

**Environment details**
 - Skupper CLI: 1.5.3
 - Skupper Operator (if applicable): N/A
 - Platform: Minkube

**Additional context**

N/A

**Comment by hash-d** [comment]
I've done some quick testing, and it seems the other flags, placed before or after `--platform` are unaffected (ie, their presence and values are respected).

**Comment by hash-d** [comment]
@fgiorgetti FYI

**Comment by hash-d** [comment]
Well...  After opening the ticket I checked the status on main, and this is actually a duplicate of #1147, already fixed on #1451 

I had not seen those before, as I searched for `--platform` instead of just `platform`.

**Comment by hash-d** [comment]
Closing this as a dup
